src/analysis/retail/evoker/preservation/normalizers/CastLinkNormalizer.ts

```diff
@@ -84,6 +160,50 @@ const EVENT_LINKS: EventLink[] = [
       );
     },
   },
+  // link seedling heal to Emerald Blossom echo heal
+  {
+    linkRelation: FLUTTERING_SEEDLINGS_ECHO,
+    reverseLinkRelation: FLUTTERING_SEEDLINGS_ECHO,
+    linkingEventId: SPELLS.EMERALD_BLOSSOM_ECHO.id,
+    linkingEventType: EventType.Heal,
+    referencedEventId: SPELLS.FLUTTERING_SEEDLINGS_HEAL.id,
+    referencedEventType: EventType.Heal,
+    anyTarget: true,
+    forwardBufferMs: EB_BUFFER_MS,
+    maximumLinks(c) {
+      return c.getTalentRank(TALENTS_EVOKER.FLUTTERING_SEEDLINGS_TALENT);
+    },
```

Comment:
Why do we need to go through all this trouble to get the specific maximum link count - isn't Fluttering Seedlings a different spellId? What are we worried about colliding with here?

Reply:
It's more that when you cast emerald blossom, it does an echo emerald blossom as well and they both spawn the seedlings, so we want to attribute the later seedlings to the echo and the first ones to the hard cast. Both types of seedlings are same spell id, which is why we need the max links.